charts/agent-control-deployment/templates/deployment-agentcontrol.yaml

```diff
@@ -116,6 +116,20 @@ spec:
 
           resources:
             {{- toYaml .Values.resources | nindent 12 }}
+          livenessProbe:
```

Comment:
Error: unable to build kubernetes objects from release manifest: error validating "": error validating data: [ValidationError(Deployment.spec.template.spec): unknown field "livenessProbe" in io.k8s.api.core.v1.PodSpec, ValidationError(Deployment.spec.template.spec): unknown field "readinessProbe" in io.k8s.api.core.v1.PodSpec]

Reply:
Is it an old error? I messed up with indentation at first but I have already updated it 😅 
I'm not sure if you are referring to the same